examples/download-local-video/sign.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
shouldn't manifest signing be generalized somehow? either by us providing a dedicated tool or handbook instructions on how to do that?

also, question: who signs the manifests? and how is this signature verified? how will a 3rd party user prepare a manifest so that the providers accept it?

Reply:
Regarding first question:
It will be described in a handbook (my next task). For now tool for signing manifest will be a plain old openssl. We could automate it if we would have e.g. tool for building yapapi apps, like lets say Cloudflare has for its worksers service (tool named wrangler). It was decided we do not want to sign manifest in yapapi directly (because it would require using private key nad its password whenever app is executed).

Reply:
Regarding second question:
`ya-provider`s will be distributed with Golem public CA certificate (and probably some intermediate certificate(s)). It will be also possible to import other certificates (certificate chains) distributed by some 3rd party authors of  apps into instance of `ya-provider` by its operators.

In case of a Golem public certificate there are two options.
It can be signed by Golem using private key owned by Golem (in such case yapapi `cert` field contains Golem public certificate).
It can be signed by owner of an certificate issued by Golem using private key of this entity (in such case `cert` field contains certificate issued by Golem for this entity)

In case of 3rd party certificate it will be signed by 3rd party private key, but it will be up to this 3rd party to convince providers to trust them and import their public CA cert into ya-provider's keystore.